app/migrations.go

```diff
@@ -257,6 +258,58 @@ func (s *Server) doGuestRolesCreationMigration() {
 	}
 }
 
+func (a *App) DoBoardRolesCreationMigration() {
+	a.Srv().doBoardRolesCreationMigration()
+}
+
+func (s *Server) doBoardRolesCreationMigration() {
+	// If the migration is already marked as completed, don't do it again.
+	if _, err := s.Store.System().GetByName(BoardRolesCreationMigrationKey); err == nil {
+		return
+	}
+
+	roles := model.MakeDefaultRoles()
+
+	allSucceeded := true
+	if _, err := s.Store.Role().GetByName(context.Background(), model.BoardAdminRoleId); err != nil {
+		if _, err := s.Store.Role().Save(roles[model.BoardAdminRoleId]); err != nil {
+			mlog.Critical("Failed to create new board admin role to database.", mlog.Err(err))
```

Comment:
Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change.